src/helpers/slack.ts

```diff
@@ -0,0 +1,108 @@
+const { WebClient, ErrorCode } = require('@slack/web-api');
+import * as slackMessageBlocks from './files/welcomeMessageBlocks.json';
+import { SlackEvent } from './types';
+const fs = require('fs');
+
+
+function getEmojisToReactWith(text: string): Array<string> {
+    const emojisToReactWith: Array<string> = [];
+
+    let rawdata = fs.readFileSync('./helpers/files/emojis.json');
+    let emojis: Record<string, Array<string>> = JSON.parse(rawdata);
+
+    for (const [emoji, keywords] of Object.entries(emojis)) {
+        for (const keyword of keywords) {
+            if (text.includes(keyword)) {
+                emojisToReactWith.push(emoji)
+            }
+        }
+    }
+
+    console.log(`Emojis to react with: ${emojisToReactWith}`)
+    return emojisToReactWith
+}
+
+async function getCurrentEmojisOnSlackPost(slackWebClient: typeof WebClient, channel: string, timestamp: string): Promise<string[]> {
+    try {
+        const reactions = await slackWebClient.reactions.get({
+            channel,
+            timestamp
+        });
+        if (reactions?.message?.reactions) {
+            // return an array of reactions
+            return reactions.message.reactions.map((reaction: Record<string, string>) => reaction.name);
```

Comment:
better to create a const var for `reactions?.message?.reactions` so that it can be reused in line 31, 33.

Reply:
is there a type inferred in `reactions`? If so, then might not need to infer for `(reaction: Record<string, string>)` inside map

Reply:
Created a constant for message reactions and also using slack provided type for reaction 👍 v